### PR TITLE
Add a resolution pack for Skylanders Giants

### PR DESCRIPTION
**Pull request by Alpha-Lin**
This pack seems to work fine with me, please tell me if anything is wrong.

### CODE REVIEW

File: Resolutions/SkylandersGiants_Resolution/rules.txt [hunk 1]
```diff
@@ -0,0 +1,111 @@
+[Definition]
+titleIds = 000500001010D700,0005000010116000
```
Comment (slashiee) [inline Resolutions/SkylandersGiants_Resolution/rules.txt:2]: Small nit: add space between the comma and the next title id

File: Resolutions/SkylandersGiants_Resolution/rules.txt [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+name = 7680x4320
+$width = 7680
+$height = 4320
+$gameWidth = 1280
+$gameHeight = 720
+
+[TextureRedefine]
+width = 1280
+height = 720
+overwriteWidth = ($width/$gameWidth) * 1280
```
Comment (slashiee) [inline Resolutions/SkylandersGiants_Resolution/rules.txt:85]: Does the game make use of videos at all? If so, I would add `tileModesExcluded = 0x001` to both the 1280x720 and 640x360 redefines.

**Review by slashiee — COMMENTED** [review]
Looks good aside from a nit and a question. I don't have the game so I can't test it.

File: Resolutions/SkylandersGiants_Resolution/rules.txt (hunk 1 above)
Comment (Alpha-Lin) [reply Resolutions/SkylandersGiants_Resolution/rules.txt]: Done.

File: Resolutions/SkylandersGiants_Resolution/rules.txt (hunk 2 above)
Comment (Alpha-Lin) [reply Resolutions/SkylandersGiants_Resolution/rules.txt:85]: Yes the game has some videos but they use other resolutions like 2048x1024 and 1024x512 and never 1280x720 or 640x360.

**Pull request merged**